services/youtube/youtube-subscribers.service.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const { documentation, YouTubeChannelBase } = require('./youtube-channel-base')
+
+module.exports = class YouTubeSubscribes extends YouTubeChannelBase {
+  static route = {
+    base: 'youtube/channel/subscribers',
+    pattern: ':channelId',
+  }
+
+  static get examples() {
+    const preview = this.render({
+      statistics: { subscriberCount: 14577 },
+      channelId: 'UCTNq28Ah5eyDgsYOA0oHzew',
```

Comment:
Could a programming/IT/coding/developer channel be used instead in all examples, including the ones in `YouTubeChannelViews`? The Apache Foundation's channel (`UCLDJ_V9KUOdOFSbDvPfGBxw`) could be a good candidate.

Our [contributing guidelines](https://github.com/badges/shields/blob/master/CONTRIBUTING.md) state that "_Shields.io hosts integrations for services which are primarily used by developers or which are widely used by developers_". A lot of our social badges are somewhat borderline, I would at least like to send the right message in the examples of said badges.

Reply:
fixesd